packages/api/src/module/write/email-sender/domain/sendEmailMessage.ts

```diff
@@ -0,0 +1,25 @@
+import {EmailMessageWasSentDomainEvent} from "@/write/email-sender/domain/events";
+import {SendEmailMessage} from "@/commands/send-email-message.domain-command";
+import {env} from "@/shared/env";
+
+export function sendEmailMessage(
+  _pastEvents: EmailMessageWasSentDomainEvent[],
+  command: SendEmailMessage,
+  appEmailAddress: env.APP_EMAIL_ADDRESS
```

Comment:
@all ktoś może zobaczyć, czemu mi tego nie łapię? Zrobiłem to wzorując się na innych importach z env, ale widocznie o czymś zapomniałem :-/ 
![image](https://user-images.githubusercontent.com/31566345/132873547-449d3961-013c-4bdb-ace4-1786b0c9c137.png)
![image](https://user-images.githubusercontent.com/31566345/132873598-dc0f8f5b-f538-40bf-8cc7-b74f08863b7f.png)

Reply:
You are trying to use value as a type

Reply:
Tutaj daj jako string, bo typ jest string. A dopiero w wywołaniu przekaz ta wartość z env :)